cfgmgr/buffer_pool_mellanox.lua

```diff
@@ -186,6 +187,20 @@ end
 -- Connect to CONFIG_DB
 redis.call('SELECT', config_db)
 
+-- Check if platform is SPC6 or later and set modification descriptors pool size
+-- Extract first digit of model number from platform string (e.g., "sn6600" -> 6, "sn6800" -> 6)
+local platform = redis.call('HGET', 'DEVICE_METADATA|localhost', 'platform')
+if platform then
+    local model_str = string.match(platform, "sn(%d)")
```

Comment:
not a problem now but for extensibility please use `string.match(platform, "sn(%d+)")` to capture one or more digits. currently, this only picks up the first digit so when it is sn10 or sn11, it'll only pick up 1.

Reply:
Exactly, thanks for the hint, code updated.